resources/views/bread/browse.blade.php

```diff
@@ -37,60 +37,60 @@
                                         <td>
                                             <?php $options = json_decode($row->details); ?>
                                             @if($row->type == 'image')
-                                                <img src="@if( strpos($data->{$row->field}, 'http://') === false && strpos($data->{$row->field}, 'https://') === false){{ Voyager::image( $data->{$row->field} ) }}@else{{ $data->{$row->field} }}@endif" style="width:100px">
+                                                <img src="@if( strpos($data->{strtolower(strtolower($row->field))}, 'http://') === false && strpos($data->{strtolower($row->field)}, 'https://') === false){{ Voyager::image( $data->{strtolower($row->field)} ) }}@else{{ $data->{strtolower($row->field)} }}@endif" style="width:100px">
```

Comment:
@ Fletch3555
This is an Oracle db issue.
Field names in the table are in upper case, but in the eloquent model they are lower case.
I just try to change some code to avoid this issue.
Thank you very much.